src/app/config.py

```diff
@@ -0,0 +1,23 @@
+"""Export global config variables."""
+import sqlfluff
+
+SQLFLUFF_VERSION = sqlfluff.__version__
+
+# manually updated on version bump
+VALID_DIALECTS = (
```

Comment:
Aside: I think you could generate this automatically, if a bit hackishly, by doing something like `[os.path.basename(p) for p in os.listdir(src.sqlfluff.core.dialects)]`.

Reply:
I considered something like that, but figured that its possible not all dialects would be registered or something. Given that new dialects are fairly rare to add, I don't feel super concerned about maintaining this list.